packages/xo-server-test/sample.config.toml

```diff
@@ -20,3 +21,6 @@
 
 [remotes]
   default = { name = '', url = '' }
+  # For delta backup, we need two remotes
+  # remote1 = { name = '', url = '' }
```

Comment:
`remote1` is required. No need the above message because all entries without hashtag are required.